packages/hoppscotch-app/components/lenses/renderers/composables/usePreview.ts

```diff
@@ -0,0 +1,35 @@
+import { ref, Ref } from "@nuxtjs/composition-api"
+
+export default function usePreview(
+  previewEnabledDefault: boolean,
+  responseBodyText: Ref<any>
+): any {
```

Comment:
Either define a concrete a concrete return type or just remove the return type definition to let TypeScript define a proper return type.